glean-core/ffi/glean.h

```diff
@@ -786,6 +786,10 @@ uint64_t glean_new_uuid_metric(FfiStr category,
                                Lifetime lifetime,
                                uint8_t disabled);
 
+int32_t glean_uuid_test_get_num_recorded_errors(uint64_t metric_id,
```

Comment:
Why is the return type signed? Are negative amounts of errors expected?

Reply:
Not sure -- but this is generated code from `cbindgen` for generated code in `macros.rs` (and thus is consistent with the other metrics).

Reply:
This is probably using a signed type because unsigned types are "experimental" in Kotlin.

Reply:
Yup, Kotlin...